Packs/OktaASA/Integrations/OktaASA/OktaASA.py

```diff
@@ -165,18 +169,28 @@ def is_token_expired(expires_date: str) -> bool:
     return current_utc_time > expires_datetime_date
 
 
-def add_time_to_events(events: List[Dict]):
+def add_time_and_related_object_data_to_events(events: List[Dict], related_objects: Dict, add_time_mapping: bool):
     """
-    Adds the _time key to the events.
+    Adds the "_time" ,"actor", "user", "client", "project" keys values to the events.
     Args:
         events: List[Dict] - list of events to add the _time key to.
+        related_objects: Dict - A dict of events related_objects to add the related objects to.
+        add_time_mapping (bool): whether to add time mapping.
     Returns:
-        list: The events with the _time key.
+        list: The events with the _time key and related object information.
     """
     for event in events:
-        create_time = arg_to_datetime(arg=event.get("timestamp"))
-        event["_time"] = create_time.strftime(DATE_FORMAT) if create_time else None
-
+        if add_time_mapping:
```

Comment:
Why? we always add the time...

Reply:
In the get-events command we do not add time.